### PR TITLE
Foxy updates

### PR DESCRIPTION
**Pull request by roni-kreinin**
- Fixed ROS2 control configuration
- Added SLAM and Nav2 launch files to `jackal_navigation`
- Cleaned up launch files for control and description

### CODE REVIEW

File: jackal_navigation/package.xml [hunk 1]
```diff
@@ -1,23 +1,21 @@
 <?xml version="1.0"?>
-<package>
+<?xml-model href="http://download.ros.org/schema/package_format3.xsd" schematypens="http://www.w3.org/2001/XMLSchema"?>
+<package format="3">
   <name>jackal_navigation</name>
-  <version>0.8.1</version>
+  <version>0.8.5</version>
```
Comment (tonybaltovski) [inline jackal_navigation/package.xml:5]: Why the version change here?

File: jackal_navigation/CHANGELOG.rst [hunk 2]
```diff
@@ -1,94 +0,0 @@
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
```
Comment (tonybaltovski) [inline jackal_navigation/CHANGELOG.rst]: Should have been deleted?

File: jackal_control/config/localization.yaml [hunk 3]
```diff
@@ -3,27 +3,28 @@ ekf_node:
...
     odom0_queue_size: 10
 
     imu0: imu/data
     imu0_config: [false, false, false,
-                  true, true, true,
                   false, false, false,
-                  true, true, true,
-                  false, false, false]
-    imu0_differential: true
+                  true, false, false,
```
Comment (tonybaltovski) [inline jackal_control/config/localization.yaml]: How come there is one true here? Isn't it X velocity which isn't measured by the IMU?

File: jackal_navigation/launch/nav2.launch.py [hunk 4]
```diff
@@ -0,0 +1,171 @@
...
+            condition=IfCondition(use_composition),
+            name='nav2_container',
+            package='rclcpp_components',
+            executable='component_container',
+            parameters=[configured_params, {'autostart': autostart}],
+            remappings=remappings,
+            output='screen'),
+
+        IncludeLaunchDescription(
+            PythonLaunchDescriptionSource(os.path.join(nav2_bringup_dir, 'launch',
```
Comment (tonybaltovski) [inline jackal_navigation/launch/nav2.launch.py]: Could `FindPackageShare` instead of `os.path.join`?

**Review by tonybaltovski — COMMENTED** [review]
Just a few questions so far but looks good!

File: jackal_navigation/package.xml (hunk 1 above)
Comment (roni-kreinin) [reply jackal_navigation/package.xml:5]: The changelogs went up to 0.8.5.

File: jackal_navigation/launch/nav2.launch.py (hunk 4 above)
Comment (roni-kreinin) [reply jackal_navigation/launch/nav2.launch.py]: Do you mean `PathJoinSubstitution`? And replace `get_package_share_directory` with `FindPackageShare`?

**Review by tonybaltovski — APPROVED** [review]

**Pull request merged**